corehq/apps/reports_core/filters.py

```diff
@@ -55,9 +58,23 @@ def get_value(self, request_params, user=None):
             kwargs.update(
                 {param.name: request_params[param.name] for param in self.params if param.name in request_params}
             )
-            return self.value(**kwargs)
+            choice_list = self.value(**kwargs)
         else:
-            return self.default_value(**kwargs)
+            choice_list = self.default_value(**kwargs)
+        if (user
+                and domain
```

Comment:
When might these not be present?  If there's some code path that doesn't provide user and domain, then that means restricted users might not have their access restricted.  If possible, I'd make these required args.

Reply:
Code block is removed now